src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,181 @@
+import duke.FileIOManager;
+
+public class TaskManager {
+    private static ArrayList<Todo> tasks = new ArrayList<Todo>();
+
+    public static void addTodo(String userInput) {
+        try {
+            if (userInput.length()<5) {
+                throw new InvalidInputException("OOPS!!! Description of todo cannot be empty :(");
+            }
+            Todo newTask = new Todo(userInput);
+            tasks.add(newTask);
+            printTaskAddedConfirmation(newTask);
+            updateFile();
+        } catch (InvalidInputException e){
+            System.out.println(e.toString().substring(23));
+        }
+    }
+
+    public static void addDeadline(String userInput) {
+        try {
+            if (userInput.length()<9){
+                throw new InvalidInputException("OOPS!!! Description of deadline cannot be empty :(");
+            }
+            if (!userInput.contains("/by ")){
+                throw new InvalidInputException("OOPS!! Please input date of deadline");
+            }
+            int startIndexOfTask = userInput.indexOf(' ')+1;
+            int endIndexOfTask = userInput.indexOf('/') - 1;
+
+            int startIndexOfDeadline = userInput.indexOf('/') + 4;
+            String taskName = userInput.substring(startIndexOfTask, endIndexOfTask);
+            String deadline = userInput.substring(startIndexOfDeadline);
+            Deadline newTask = new Deadline(taskName, deadline);
+            tasks.add(newTask);
+            printTaskAddedConfirmation(newTask);
+            updateFile();
+        } catch (InvalidInputException e){
+            System.out.println(e.toString().substring(23));
+        }
+    }
+
+    public static void addEvent(String userInput) {
+        try {
+            if (userInput.length()<6) {
+                throw new InvalidInputException("OOPS!!! Description of event cannot be empty :(");
+            }
+            if (!userInput.contains("/at ")){
+                throw new InvalidInputException("OOPS!! Please input time of event");
+            }
+            int startIndexOfTask = userInput.indexOf(' ')+1;
+            int endIndexOfTask = userInput.indexOf('/') - 1;
+            int startIndexOfDate = userInput.indexOf('/') + 4;
+            String taskName = userInput.substring(startIndexOfTask, endIndexOfTask);
+            String deadline = userInput.substring(startIndexOfDate);
+            Event newTask = new Event(taskName, deadline);
+            tasks.add(newTask);
+            printTaskAddedConfirmation(newTask);
+            updateFile();
+        } catch (InvalidInputException e){
+            System.out.println(e.toString().substring(23));
+        }
+
+    }
+
+    public static void printTasks() {
+        if (tasks.size() == 0) {
+            System.out.println("\tNo tasks added");
+            return;
+        }
+        int count = 1;
+        for (Todo task : tasks) {
+            System.out.println("\tHere are the tasks in your list");
```

Comment:
Such print statements can be extracted out as constants to prevent repeating yourself in line with the DRY (Don't Repeat Yourself) principle